index.html

```diff
@@ -1,197 +1,274 @@
 
-          </ul>
-        </article>
-        <article class="professional-card">
-          <img src="assets/images/icon-skills.svg" alt="" />
-          <h3 class="professional-card-title">Skills</h3>
-          <ul class="professional-card-list">
-            <li class="professional-list-item">Codekit</li>
-            <li class="professional-list-item">GitHub</li>
-            <li class="professional-list-item">Codepen</li>
-            <li class="professional-list-item">Gitlab</li>
-            <li class="professional-list-item">Terminal</li>
-          </ul>
-        </article>
-      </section>
     </div>
-  </body>
-</html>
+     <div class="about-content">
+      <article class="professional-card">
+        <img src="assets/images/icon-languages.svg" alt="" />
+        <h3 class="professional-card-title">Languages</h3>
+        <ul class="professional-card-list">
+          <li class="professional-list-item">javScript</li>
+          <li class="professional-list-item">Ruby</li>
+          <li class="professional-list-item">Html</li>
+          <li class="professional-list-item">Css</li>
+        </ul>
+      </article>
+      <article class="professional-card">
+        <img src="assets/images/icon-frameworks.svg" alt="" />
+        <h3 class="professional-card-title">Frameworks</h3>
+        <ul class="professional-card-list">
+          <li class="professional-list-item">Bootstrap</li>
+          <li class="professional-list-item">Ruby On Rails</li>
+          <li class="professional-list-item">Rspsec</li>
+          <li class="professional-list-item">Capybara</li>
+          <li class="professional-list-item">Selenium</li>
+
+        </ul>
+      </article>
+      <article class="professional-card">
+        <img src="assets/images/icon-skills.svg" alt="" />
+        <h3 class="professional-card-title">Skills</h3>
+        <ul class="professional-card-list">
+          <li class="professional-list-item">Codekit</li>
+          <li class="professional-list-item">GitHub</li>
+          <li class="professional-list-item">Codepen</li>
+          <li class="professional-list-item">Gitlab</li>
+          <li class="professional-list-item">Terminal</li>
+        </ul>
+      </article>
+    </div>
+    </section>
+  <section class="contact-me" id="contact">
+    <p class="contact-description">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/meqnzwlb" method="POST" class="contact-form">
+      <input type="text" name="name" id="name" placeholder="Full name" class="form-field" maxlength="30" required>
+      <input type="email" name="email" id="email" placeholder="Email address" class="form-field" maxlength="30"
+        required>
+      <textarea class="contact-message" maxlength="500" placeholder="Enter text here" required></textarea>
+      <input type="submit" value="Get in touch" id="submit" class="button">
```

Comment:
- [ ] OPTIONAL: I recommend adding background images in contact section to make your page look even better